db/sample_data/events.rb

```diff
@@ -124,3 +124,24 @@ def event_bachlorpodium
       published: true
   )
 end
+
+def event_summer_camp
+  date_range_summer_camp = DateRange.create!(
+      start_date: Date.new(2017, 06, 30),
+      end_date: Date.new(2017, 07, 04)
+  )
+
+  Event.new(
+      name: 'Sommer-Camp',
+      description: 'Jeder hat schon einmal eine Projektarbeit alleine oder in einem kleinen Team durchgeführt. Wie geht man jedoch vor, wenn an ein Projekt mit 39 Kolleginnen und Kollegen bearbeitet? Beim Sommercamp lernen die Teilnehmenden dies an einem praktischen Beispiel. Neben der gemeinsamen Lösung von Teilproblemen stehen dabei Themen wie effiziente Aufgabenteilung, effektive Kommunikation und Projektorganisation auf dem Plan. Programmiererfahrungen in einer objektorientierten Sprache sind hilfreich, jedoch nicht nötig. Im Vordergrund steht die Bereitschaft, engagiert zur Teamarbeit beizutragen sowie das Interesse am Lösen algorithmischer Problemstellungen.​ Neben diesen inhaltlichen Schwerpunkten haben die Schüler auch Gelegenheit, das Campusleben am HPI kennen zu lernen, Erfahrungen auszutauschen und neue Kontakte zu knüpfen. Sie werden während des gesamten Camps von Studenten des HPI betreut. **Kosten:** Teilnehmergebühr: 100 EUR Reisekostenanteil: 20 EUR ',
+      max_participants: 25,
+      organizer: 'HPI Schülerklub',
+      knowledge_level: 'Fortgeschrittene',
+      date_ranges: [date_range_summer_camp],
+      application_deadline: Date.tomorrow,
+      application_status_locked: false,
```

Comment:
This field has recently been split into two and is the reason why the tests fails.